apps/fabric-website/src/pages/Styles/Fabric7Page/docs/web/Fabric7Overview.md

```diff
@@ -1,15 +1,15 @@
-### What’s new in Fabric 7?
+### What’s new in Fluent UI 7?
 
-We’ve updated Fabric to include a new set of Web Fluent design styles, which include colors, type, depth and motion which are applied to the standard set of Fabric controls. These include:
+We’ve updated Fluent UI to include a new set of Web Fluent design styles, which include colors, type, depth and motion which are applied to the standard set of Fluent UI controls. These include:
```

Comment:
Can we add a note here about how Fluent UI 7 is the same as OUFR 7 (and is what you should upgrade to from OUFR 6)?